common/src/main/java/com/datastax/oss/simulacron/common/cluster/QueryLog.java

```diff
@@ -97,6 +98,11 @@ public QueryLog(
           }
         }
       }
+    } else if (frame.message instanceof Prepare) {
+      Prepare prepare = (Prepare) frame.message;
+      this.consistency = ConsistencyLevel.ANY;
```

Comment:
In this case it's ok to leave consistency/serialConsistency as unset since it is not applicable for prepare.